packages/aws-cdk-lib/aws-rds/test/cluster.test.ts

```diff
@@ -209,6 +209,30 @@ describe('cluster new api', () => {
         ],
       });
     });
+
+    test('preferredMaintenanceWindow provided in InstanceProps', () => {
+      // GIVEN
+      const stack = testStack();
+      const vpc = new ec2.Vpc(stack, 'VPC');
+
+      const PREFERRED_MAINTENANCE_WINDOW: string = 'Sun:12:00-Sun:13:00';
```

Comment:
We should test when the property is not using the format. It will likely to happen because of a typo.